vendor/bat-native-ledger/src/bat/ledger/internal/response/response_publisher.h

```diff
@@ -0,0 +1,23 @@
+/* Copyright (c) 2020 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVELEDGER_COMMON_RESPONSE_PUBLISHER_H_
+#define BRAVELEDGER_COMMON_RESPONSE_PUBLISHER_H_
+
+#include <memory>
+#include <string>
+
+#include "bat/ledger/mojom_structs.h"
+
+namespace braveledger_response_util {
+
+ledger::ServerPublisherInfoPtr ParsePublisherInfoResponse(
```

Comment:
I would prefer not merge any refactors of the publist code at this time (as there might be uplift requirements).